### ISSUE TITLE
"Don't know how to build task 'test'"

### ISSUE THREAD

**Issue by mroach** [renchap/webpacker-react#33]
https://github.com/renchap/webpacker-react/blob/master/lib/tasks/example_app.rake#L15

This line requires that there be a rake task called `test` defined. If none is defined in your app, the entire rake system is broken. Can't run `rake -T` or any tasks.

Was this meant to be done in the *example_app* namespace?

My `Rakefile`

```
require_relative 'config/application'

Rails.application.load_tasks
```

Using Ruby 2.4.1, Rails 5.1, webpacker 2.0, webpacker-react 0.3.0

**Comment by renchap** [comment]
Yes, I added this to ensure that `example_app:webpack` was launched before running this gem tests.

From what I understand, in your case this task is also launched when you run `rails test` in your Rails project? It is not the case for my project using Rails 5.1 (but I have the standard `test` rake task created by Rails).

Do you have any idea why a task defined in the `webpacker-react ` gem is loaded by rake in your own project?

**Comment by mroach** [comment]
In my project I'm not using MiniTest so there's no `test` task, only `spec` for RSpec. I'm guessing that `Rails.application.load_tasks` eagerly loads all tasks from all gems, right?

**Comment by renchap** [comment]
This is fixed in 0.3.1, thanks for the report!
It looks like Rails indeed loads this path for all gems, I did not knew this.
